internal/policy/operator/testdata/disconnected_bundle/metadata/annotations.yaml

```diff
@@ -5,7 +5,7 @@ annotations:
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: simple-disconnected-operator
   operators.operatorframework.io.bundle.channels.v1: alpha
-  operators.operatorframework.io.metrics.builder: operator-sdk-v1.32.0
+  operators.operatorframework.io.metrics.builder: operator-sdk-v1.33.0
```

Comment:
@tkrishtop We have more test data now, could you also update those bundles in `/testdata` as well?

Reply:
hi @acornett21 that's fixed and the tests are green, could you please have a look one more time?

Reply:
I still only see this file committed, shouldn't there be other test files?